helpers/dateTime.js

```diff
@@ -161,6 +161,44 @@ export function getUTCDateTimeFromLocalDateTime(date, time) {
 	return formatDateTimeInISO(utcDateTime);
 }
 
+export function getUTCDateTimeRange(type, diff) {
+	if (!type || (!diff && diff !== 0)) return {};
+	if (type !== 'hours' && type !== 'days' && type !== 'weeks' && type !== 'months') return {};
```

Comment:
Should this helper support more timeframes from the get-go? Thinking of `years`, `minutes`, `seconds`.